### PR TITLE
add packages needed for heavy-sync

### PR DESCRIPTION
**Pull request by phunehehe**

### CODE REVIEW

File: pkgs/boto/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ pkgs ? import <nixpkgs> {}, ... }:
```
Comment (proger) [inline pkgs/boto/default.nix]: btw i think it would be nice to move python stuff to `pkgs/python/{boto,...}` because it's getting crowded

**Comment by phunehehe** [comment]
@proger I found that I didn't need all the other packages, they are already in nixpkgs. The change in to-nix/python/python2nix/**main**.py still seems useful so I'd keep it.

**Comment by proger** [comment]
@phunehehe so ready to merge?

**Comment by phunehehe** [comment]
yep

**Pull request merged**